index.html

```diff
@@ -1,16 +1,109 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Responsive Cake webpage</title>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link
+      rel="icon"
+      href="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSZMAPmi5t1WxKX3-RIVRwvOyeqs9GWxDWKBGGRachiwSQsizztKaWuU1v-RTQVu9-zrCE&usqp=CAU"
+      type="image/x-icon"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Architects+Daughter&family=Fuzzy+Bubbles&family=M+PLUS+1p:wght@300&display=swap"
+      rel="stylesheet"
+    />
+    <title>CHILDREN’S BIRTHDAY CAKES</title>
     <!-- Add a link to your css file here -->
-</head>
-
-<body>
-    <!-- Add your markup here -->
-</body>
-
-</html>
\ No newline at end of file
+    <link rel="stylesheet" href="./css/style.css" />
+    <link rel="stylesheet" href="./css/header.css" />
+    <link rel="stylesheet" href="./css/footer.css" />
```

Comment:
Great work Mahri, I like your approach of creating different style sheets for different sections. Well done!